android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt

```diff
@@ -56,9 +56,12 @@ class RNCWebViewManagerImpl {
     private val DEFAULT_LACK_PERMISSION_TO_DOWNLOAD_MESSAGE =
         "Cannot download files as permission was denied. Please provide permission to write to storage, in order to download files."
 
+    fun createRNCWebViewInstance(context: ThemedReactContext): RNCWebView {
+        return RNCWebView(context)
+    }
```

Comment:
I'm not sure this will help unless you also provide a similar method to substitute the `RNCWebViewManagerImpl` itself used by `RNCWebViewManager.java`...?

(That's why I was wondering if it would make sense for `createViewInstance` to take an optional `RNCWebView` argument instead... but, subclassing the whole `RNCWebViewManagerImpl` would also probably work... or maybe you have another approach in mind)

Reply:
Yeah in my mind you would subclass the impl